chaoscenter/web/src/views/ChaosInfrastructureReferenceField/ChaosInfrastructureReferenceField.module.scss

```diff
@@ -190,3 +198,37 @@
     }
   }
 }
+
+.listEnvContainer {
+  background: #effbff;
+  box-shadow: 0px 0px 1px rgba(40, 41, 61, 0.04), 0px 2px 4px rgba(96, 97, 112, 0.16);
+  border-radius: 8px !important;
+  cursor: pointer;
+}
+
+.itemEnv {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 3fr 25px;
```

Comment:
i don't think 3fr makes any diff, since the other values is a constant 3fr is same as 1fr